coverconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "enabled": true,
+    "relativeSourcePath": "..",
+    "relativeCoverageDir": "../coverage",
+    "ignorePatterns": ["**/node_modules/**"],
+    "includePid": false,
+    "reports": ["json", "html", "lcov"],
+    "verbose": false
+}
```

Comment:
I'm a bit surprised a codecov config is needed?
What necessitates it?

Reply:
It probably isn't needed, I'm just trying to follow [this guide](https://github.com/codecov/example-typescript-vscode-extension) as close as possible.